test/integration/apiserver/oidc/oidc_test.go

```diff
@@ -941,6 +944,7 @@ func startTestAPIServerForOIDC[L utilsoidc.JosePublicKey](t *testing.T, oidcURL,
 			fmt.Sprintf("--oidc-client-id=%s", oidcClientID),
 			fmt.Sprintf("--oidc-ca-file=%s", oidcCAFilePath),
 			fmt.Sprintf("--oidc-username-prefix=%s", defaultOIDCUsernamePrefix),
+			"--oidc-username-claim=user",
```

Comment:
Hmm would be good to have at least one test that uses the default flag value.

Reply:
Modified just the one test (`ID token is ok`) to validate with `user` claim and retained the rest of tests to use the `sub` claim as before.